src/components/ProfileCard/ProfileCard.stories.tsx

```diff
@@ -0,0 +1,374 @@
+import { Box } from '@mui/material';
+
+import { ProfileCard } from './ProfileCard';
+import { PROFILE_CARD_SIZES, PROFILE_CARD_VARIANTS } from './ProfileCard.constants';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta = {
+  title: 'Components/ProfileCard',
+  component: ProfileCard,
+  parameters: {
+    layout: 'centered',
+    docs: {
+      description: {
+        component:
+          'A reusable profile card component for displaying user information and statistics.',
+      },
+    },
+  },
+  argTypes: {
+    name: {
+      control: 'text',
+      description: 'User full name',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    username: {
+      control: 'text',
+      description: 'User username/handle',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    avatarUrl: {
+      control: 'text',
+      description: 'URL for the user avatar image',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'undefined' },
+      },
+    },
+    role: {
+      control: 'text',
+      description: 'User role or title',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'undefined' },
+      },
+    },
+    size: {
+      control: 'select',
+      options: Object.keys(PROFILE_CARD_SIZES),
+      description: 'Size variant of the card',
+      table: {
+        type: { summary: 'small | medium | large' },
+        defaultValue: { summary: 'medium' },
+      },
+    },
+    variant: {
+      control: 'select',
+      options: Object.keys(PROFILE_CARD_VARIANTS),
```

Comment:
Same issue as above for variants: `Object.keys(PROFILE_CARD_VARIANTS)` returns keys but you want the variant values.

**Sugerencia:**
```typescript
Use `Object.values(PROFILE_CARD_VARIANTS)` instead:

options: Object.values(PROFILE_CARD_VARIANTS),
```